man/systemd.service.xml

```diff
@@ -1157,6 +1157,27 @@
         cgroup associated with the service.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>OpenFile=</varname></term>
+        <listitem><para>Takes an argument of the form <literal>path:fd-name:options</literal>,
```

Comment:
hmm, i'd always have assumed that the orer would be name:path:options

Reply:
Given that `name` and `options` are optional, it's probably better to start with the `path`.